geomesa-web/geomesa-web-data/pom.xml

```diff
@@ -44,7 +44,7 @@
             <dependency>
                 <groupId>org.apache.commons</groupId>
                 <artifactId>commons-vfs2</artifactId>
-                <version>2.0</version>
+                <version>2.1</version>
```

Comment:
Should we externalize the a commons-vfs2.version?

Reply:
we should probably just get rid of the spark endpoint tbh. i think that's the only reason this is here... it was causing unit test failures in this module with the older version.

Reply:
Let's nuke it!

Reply:
opened a ticket here: https://geomesa.atlassian.net/browse/GEOMESA-2210